source/switching_to_live/set_up_a_live_smartpay_account/index.html.md.erb

```diff
@@ -152,7 +154,9 @@ Complete the following fields on this page:
 
 5. Select the test transaction and check if you can refund it.
 
-> The refund option may take up to 20 minutes to appear after submitting the transaction.
+    The refund option may take up to 20 minutes to appear after submitting the transaction.
+
+    If you have not [set up notifications](#configure-server-communication-on-smartpay) correctly, some features such as refunds will not be available.
```

Comment:
I wonder if this could be more active and closely related to the paragraph above e.g “If the refund option does not appear, check you have set up notifications correctly.”

Also, we need notifications to be set up correctly for other reasons. I’m a little worried a service user could think, “I don’t need to do refunds so I’m not going to bother setting up notifications.”

Reply:
agree, will amend